tensorflow/lite/micro/kernels/pooling_test.cc

```diff
@@ -496,7 +496,7 @@ TF_LITE_MICRO_TEST(SimpleAveragePoolTestInt8PaddingSameStride1ActNone) {
        F2QS(8.5, output_min, output_max), F2QS(7., output_min, output_max)},
       {4, 1, 2, 4, 1},         // Output shape
       output_min, output_max,  // output quantization range
-      kTfLitePaddingValid, kTfLiteActNone, output_data);
+      kTfLitePaddingSame, kTfLiteActNone, output_data);
```

Comment:
What was the reason for this test change?

Reply:
If you look at line 474 (a little above) you will see that the test name is *SimpleAveragePoolTestInt8PaddingSameStride1ActNone*, which actually isn’t consistent with this line. Inside ops implementation we also use this field to get a padding values.